artsy-auth.gemspec

```diff
@@ -15,13 +17,17 @@ Gem::Specification.new do |s|
 
   s.files = Dir['{app,config,lib}/**/*', 'Rakefile', 'README.md']
 
-  s.add_dependency 'omniauth-oauth2'
   s.add_dependency 'omniauth-artsy', '>= 0.2.2'
```

Comment:
Will bump up required version once https://github.com/artsy/omniauth-artsy/pull/16 is done.